### DISCUSSION TITLE
user doesn't have any roles or permissions assigned

### DISCUSSION THREAD

**Discussion by mvasicek** [spatie/laravel-permission#1985] [Q&A]
Hi, I struggle with fetching roles and permissions of the user, that are clearly seeded in the database, but all User::with('permissions')->find(1), User::with('roles')->find(1) return empty arrays. I don't use any gates, only teams option with foreign key groups_id.

**Comment by erikn69** [comment]
Read documentation [working-with-teams-permissions](https://spatie.be/index.php/docs/laravel-permission/v5/basic-usage/teams-permissions#working-with-teams-permissions), you need `setPermissionsTeamId($your_team_id);` before using anything related with roles with teams, also you must read related discussions first 
Maybe helps https://github.com/spatie/laravel-permission/discussions/1906#discussioncomment-1584335
